helpers/helper-translation-discussion.php

```diff
@@ -251,6 +252,13 @@ public function rewrite_original_post_type_permalink( string $post_link, WP_Post
 		return $cache[ $post->ID ];
 	}
 
+	public function comments_open_override( $open, $post_id ) {
```

Comment:
Can you add the header for this function?

Reply:
Thank you, added.